scripts/print_keys

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+erl -pa {{lib_dirs}}/**/ebin -noshell -eval 'miner_keys:print_keys("{{sys_config}}")' -eval 'init:stop()'
```

Comment:
Why not use a miner subcommand that doesn't do a rpc call?

Reply:
Because I was lazy and this was quicker?

Reply:
I’ll get that fixed up